chat.go

```diff
@@ -348,8 +334,8 @@ type responseURLSender struct {
 	deleteOriginal  bool
 }
 
-func (t responseURLSender) BuildRequest() (*http.Request, func(*chatResponseFull) responseParser, error) {
-	req, err := jsonReq(t.endpoint, Msg{
+func (t responseURLSender) BuildRequest(ctx context.Context) (*http.Request, func(*chatResponseFull) responseParser, error) {
```

Comment:
[NOTE]
`BuildRequest` is public method.
However users cannot use this directly.
So this changes maintain backward compatibility.